crates/uv-client/tests/user_agent_version.rs

```diff
@@ -197,25 +194,18 @@ async fn test_user_agent_has_linehaul() -> Result<()> {
     if cfg!(windows) {
         assert_eq!(linehaul.distro, None);
     } else if cfg!(target_os = "linux") {
-        // Using `os_info` to confirm our values are as expected in Linux
-        let info = os_info::get();
         let Some(distro_info) = linehaul.distro else {
             panic!("got no distro, but expected one in linehaul")
         };
-        assert_eq!(distro_info.id.as_deref(), info.codename());
-        if let Some(ref name) = distro_info.name {
-            assert_eq!(name, &info.os_type().to_string());
-        }
-        if let Some(ref version) = distro_info.version {
-            assert_eq!(version, &info.version().to_string());
-        }
+        assert_eq!(distro_info.id.as_deref(), Some("jammy"));
+        assert_eq!(distro_info.name.as_deref(), Some("Ubuntu"));
+        assert_eq!(distro_info.version.as_deref(), Some("22.04"));
```

Comment:
This breaks running the tests locally

Reply:
Really? This should be pulled from the platform information we're passing directly right?

Reply:
For linux we're not passing the information in we're using the actual platform (`sys_info::linux_os_release()`)

```
thread 'test_user_agent_has_linehaul' panicked at crates/uv-client/tests/user_agent_version.rs:200:9:
assertion `left == right` failed
  left: Some("mantic")
 right: Some("jammy")
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

(mantic is what i'm running locally)

Reply:
I see we can't pass mock data in... uhh hmm.

Reply:
This should hopefully be fixed in https://github.com/astral-sh/uv/pull/2923 since it uses `sys_info::linux_os_release()` now, which is what the actual linehaul code uses.